docs/gettingstarted/installation-macos.rst

```diff
@@ -121,6 +118,7 @@ Installation
 
    .. code-block:: console
 
+    $ createdb saleor
```

Comment:
Could you move it to its own section (as `6. Create the database`)?

Reply:
I'd not create the database until the role exists as you'd normally pass the role as `--owner` to `createdb`.